Realm.podspec

```diff
@@ -21,25 +22,24 @@ Pod::Spec.new do |s|
   s.documentation_url       = "http://realm.io/docs/cocoa/#{s.version}"
   s.license                 = { :type => 'Apache 2.0', :file => 'LICENSE' }
 
-  public_header_files = 'include/Realm/RLMArray.h',
-                        'include/Realm/RLMCollection.h',
-                        'include/Realm/RLMConstants.h',
-                        'include/Realm/RLMListBase.h',
-                        'include/Realm/RLMMigration.h',
-                        'include/Realm/RLMObject.h',
-                        'include/Realm/RLMObjectBase.h',
-                        'include/Realm/RLMObjectSchema.h',
-                        'include/Realm/RLMPlatform.h',
-                        'include/Realm/RLMProperty.h',
-                        'include/Realm/RLMRealm.h',
-                        'include/Realm/RLMResults.h',
-                        'include/Realm/RLMSchema.h',
-                        'include/Realm/Realm.h'
+  public_header_files       = 'include/Realm/RLMArray.h',
+                              'include/Realm/RLMCollection.h',
+                              'include/Realm/RLMConstants.h',
+                              'include/Realm/RLMListBase.h',
+                              'include/Realm/RLMMigration.h',
+                              'include/Realm/RLMObject.h',
+                              'include/Realm/RLMObjectBase.h',
+                              'include/Realm/RLMObjectSchema.h',
+                              'include/Realm/RLMPlatform.h',
+                              'include/Realm/RLMProperty.h',
+                              'include/Realm/RLMRealm.h',
+                              'include/Realm/RLMResults.h',
+                              'include/Realm/RLMSchema.h',
+                              'include/Realm/Realm.h'
 
   s.compiler_flags          = "-DTIGHTDB_HAVE_CONFIG -DREALM_SWIFT=0 -DREALM_VERSION='@\"#{s.version}\"'"
```

Comment:
`RealmSwift.podspec` should be set its own compiler flags. Just `-DREALM_SWIFT=1` I think.